metplotpy/contributed/weather_regime/plot_weather_regime.py

```diff
@@ -0,0 +1,99 @@
+import matplotlib
+matplotlib.use('agg')
+import matplotlib.pyplot as plt
+import matplotlib.colors as colors
+from pylab import *
+import cmocean
+from cartopy.mpl.gridliner import LONGITUDE_FORMATTER, LATITUDE_FORMATTER
+import cartopy.crs as ccrs
+import cartopy.feature as cfeature
+from cartopy.util import add_cyclic_point
+import numpy as np
+import sys
+sys.path.insert(0, "/glade/u/home/kalb/UIUC/METcalcpy/")
```

Comment:
path is cheyenne-specific, use relative path with the appropriate number of '../' to get to intended directory

Reply:
This should now be fixed